src/workerd/api/BUILD.bazel

```diff
@@ -23,6 +23,7 @@ filegroup(
             "rtti.c++",
             "url.c++",
             "util.c++",
+            "message-channel.c++",
```

Comment:
Should this target reference the "message-channel" dependency instead of pulling in the files itself?

Reply:
This is in `excludes` array

Reply:
ah! missed that :-)